package.json

```diff
@@ -40,9 +41,9 @@
     "@types/jest": "25.1.4",
     "@types/node": "13.9.1",
     "babel-jest": "25.1.0",
-    "bs-platform": "9.0.0",
-    "gentype": "3.42.0",
+    "gentype": "^4.1.0",
```

Comment:
```suggestion
    "gentype": "4.1.0",
```

Reply:
Why ?

Reply:
to be sure that the installed version will be strictly the one defined in the package.json
and avoid being on the wrong version by mistake

Reply:
Okay I guessed it... so do the others ?

Reply:
yes !